docs/navigate.rst

```diff
@@ -80,3 +80,126 @@ Refresh Visibility tab
+         
+         a. Dashboard displays a **Telemetry Consumer** drop-down list. Select the appropriate consumer from which the statistics need to be obtained. The Virtual pool statistics will be displayed on the dashboard including Bits, Packets, Connections and Requests. 
+         
+            .. note::
+         
+               - Telemetry plugin needs to be installed on the BIG-IP to be able to view the Telemetry consumer list and stats from the telemetry consumer. Check the **Configure Telemetry** section for installation and configuration.
+            
+               - If there is only a single Telemetry pull consumer configured, then FASC will directly query that consumer and display the stats.
+            
+         b. **View Stats** - Dashboard displays a **View Stats** link besides the default pool. It displays total pool member count and the availability and enabled state of the pool  members using a pie chart.
+         
+         c. **View Logs** - Dashboard displays a **View Logs** link besides the VIP, which, on click will open a popup window and display all the logs for the selected Virtual Server. 
+         
+            .. note::
+
+               - It is possible to also view all logs instead of the filtered logs.
+            
+               - It is possible to specify a UTC from and to Date/Time for the logs to be retrieved. The From/To time should be specified in UTC timezone. The response too will be in UTC.
+
+         d. **View Connections** - Dashboard will display a **View Connections** link which, on click will display all the active connections to that VIP.
+         
+         e. **APIC Endpoint Details** - Visibility dashboard displays APIC details for the VIP:
+                                        1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap
+                                        
+         f. **BIG-IP Endpoint Details** - Visibility dashboard displays BIG-IP details for the VIP:
+                                          1. MAC 2. VLAN 3. Interfaces (There is a **View Logs** link besides interfaces to view the interface logs) 4. Self IPs
+
+
+View Node Dashboard
+-------------------
+
+1. Click the Visibility tab, and then from the Table list, click the Node table.
+
+2. From the Partition list, click the partition of interest.
+
+3. The Node table for this BIG-IP and partition will be displayed.
+
+4. Click the Node for which the dashboard is to be displayed. This redirects to the Visibility Dashboard sub-tab. 
+
+         .. note::
+   
+            - It is possible to directly click the **Visibility Dashboard** sub-tab and then select the Node, instead of a redirect from **Visibility Table**.
+      
+5. Visibility Dashboard displays the information for the selected VIP including Node name, route domain and monitor.
+
+         a. **View Logs** - Dashboard displays a **View Logs** link besides the Node, which, on click will open a popup and all the logs for the selected Node.
```

Comment:
The Dashboard displays.....  link next to the Node, which, when clicked opens a popup...